app/services/task_retry_service.rb

```diff
@@ -0,0 +1,41 @@
+class TaskRetryService < TaskService
+  LOST_TIME_INTERVAL = ClowderConfig.instance["LOST_TIME_INTERVAL"].to_i * 60
+
+  def process
+    interrupted_tasks.each do |task|
+      retry_task = LaunchJobTask.create!(task_options(task))
```

Comment:
@hsong-rh We should another Task type in app.models called RetryTask so they can be created and tracked separate from the LaunchJobTask.